File: video/conversations/invite-participant/invite-participant.m
NSError *error = nil;

// See the "Create a Conversation" guide for instructions on configuring a TWCConversationsDelegate
BOOL didInvite = [self.conversation invite:@"charles" error:&error];
if(didInvite) {
	NSLog(@"Invite sent!");
} else {
	NSLog(@"Unable to send the Invite");
}
/* Results in a callback to
TWCConversationDelegate#conversation:didConnectParticipant or
TWCConversationDelegate#conversation:didFailToConnectParticipant */

File: video/conversations/invite-participant/invite-participant.java
Set<String> newParticipants = new HashSet<>();
newParticipants.add("charles");

/* See the "Create a Conversation" guide for instructions on configuring a
ConversationListener */
conversation.invite(newParticipants); 
/* Results in a callback to ConversationListener#onParticipantConnected or
ConversationListener#onFailedToConnectParticipant */
